open_city_profile/tests/settings.py

```diff
@@ -2,6 +2,8 @@
 
 USE_X_FORWARDED_FOR = True
 
+INSTALLED_APPS += ["open_city_profile.tests.app"]  # noqa
```

Comment:
I think it would be good to document what check the `noqa` comment is concerning. I guess in this case it's F405. So the comment would be `# noqa: F405`.

Reply:
That style is used in 2/11 cases in the project. But I can add it here.

Reply:
Yep, it's not very common. Burden from the olden times.